GsaGH/Components/0_Model/GetGeometry.cs

```diff
@@ -1,92 +1,231 @@
 ﻿using System;
 using System.Collections.Concurrent;
 using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Drawing;
 using System.Linq;
 using System.Threading.Tasks;
 using System.Windows.Forms;
 using GH_IO.Serialization;
 using Grasshopper;
 using Grasshopper.Kernel;
+using Grasshopper.Kernel.Data;
 using Grasshopper.Kernel.Types;
 using GsaAPI;
-using GsaGH.Parameters;
 using GsaGH.Helpers.GH;
+using GsaGH.Helpers.Graphics;
+using GsaGH.Helpers.Import;
+using GsaGH.Parameters;
+using GsaGH.Properties;
 using Newtonsoft.Json;
+using OasysGH;
+using OasysGH.Components;
 using OasysGH.Parameters;
+using OasysGH.UI;
 using OasysGH.Units;
 using OasysGH.Units.Helpers;
 using OasysUnits;
 using OasysUnits.Serialization.JsonNet;
 using OasysUnits.Units;
+using Rhino.Display;
 using Rhino.Geometry;
-using System.Collections.ObjectModel;
-using Newtonsoft.Json.Linq;
-using OasysGH.Components;
-using OasysGH;
 
-namespace GsaGH.Components
-{
-    /// <summary>
-    /// Component to retrieve geometric objects from a GSA model
-    /// </summary>
-    public class GetGeometry : GH_OasysTaskCapableComponent<GetGeometry.SolveResults>, IGH_PreviewObject, IGH_VariableParameterComponent
-  {
+namespace GsaGH.Components {
+  /// <summary>
+  ///   Component to retrieve geometric objects from a GSA model
+  /// </summary>
+  public class GetGeometry : GH_OasysTaskCapableComponent<GetGeometry.SolveResults>,
```

Comment:
why static methods before fields/properties?

Reply:
will change the orger after fixing bugs